src/common/vars.ts

```diff
@@ -42,5 +42,8 @@ export const apiKubePrefix = "/api-kube"; // k8s cluster apis
 // Links
 export const issuesTrackerUrl = "https://github.com/lensapp/lens/issues";
 export const slackUrl = "https://join.slack.com/t/k8slens/shared_invite/enQtOTc5NjAyNjYyOTk4LWU1NDQ0ZGFkOWJkNTRhYTc2YjVmZDdkM2FkNGM5MjhiYTRhMDU2NDQ1MzIyMDA4ZGZlNmExOTc0N2JmY2M3ZGI";
-export const docsUrl = "https://docs.k8slens.dev/";
 export const supportUrl = "https://docs.k8slens.dev/latest/support/";
+
+const docsVersion = isProduction ? `v${packageInfo.version}` : "latest";
```

Comment:
Doesn't work for releases like 4.1.0-beta.2 but probably no big deal?

Reply:
A good catch, probably a good idea to strip the suffix